firmware/util/efilib.h

```diff
@@ -45,19 +45,6 @@ int indexOf(const char *string, char ch);
 float atoff(const char *string);
 int atoi(const char *string);
 
-#if defined(__cplusplus) && defined(__OPTIMIZE__)
-#include <type_traits>
```

Comment:
how is this related to "Debuggability in the new sensor world"? :)

Reply:
see second bullet point:
> Fix efilib.h - it didn't compile if it was the first/only thing included in a file, which somehow flew under the radar for a long time.